_contentTemplates/common/get-started.md

```diff
@@ -89,7 +48,7 @@ To prepare the project for the Telerik UI for Blazor components, install the `Te
     
         <TelerikButton>Say Hello</TelerikButton>
         
-1. Optionally, hook up a click handler that will show a message. The resulting view should look like this:
+1. Optionally, hook up a click handler that will show a message. The resulting view will look like this:
```

Comment:
Let's make this step simpler and add `OnClick` right from the start. Buttons that don't do anything make no sense anyway.

Reply:
We haven't received reports from customers about difficulties in this step, but OK.